### PR TITLE
add BCN file to class repo

### LINKED ISSUE #22: ruthtubau tracks progress on GitHub Flow
## GitHub Flow
- [x] create a **branch** (git)
- [x] add some commits (git)
- [ ] open a **pull request** (GitHub)
- [ ] additional **commits** (git)
- [ ] **merge** (git)

### PR DESCRIPTION
**Pull request by ruthtubau**
This Pull Request (PR) adds BCN file to the class repo. 
This PR closes #22 

- @iservan what's your favorite beach in Barcelona? 😎 
- @CarlosVendo do you know any good clubs in Barcelona? 👯  
- What's your favorite attraction in Barcelona?

### CODE REVIEW

**Comment by Albertovs** [comment]
@ruthtubau, Anchovies and ham? :0

**Comment by iservan** [comment]
Hi @ruthtubau I like to go to Castelldefels beach 🐚 🌊 🐟

**Comment by sebastianvendo** [comment]
@CarlosVendo goes to Razzmatazz every week. It must be good.

File: ruthtubau-BCN.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Barcelona, Catalunya
+
+### Restaurants
+
+### Attractions for Tourists
+
+- Sagrada Familia
+- Parc Guell
+- Casa Batllo
```
Comment (iservan) [inline ruthtubau-BCN.md]: I've heard Casa Batllo is is the master piece of Gaudi 🏠 💸

**Comment by ruthtubau** [comment]
I've added your suggestions @iservan @sebastianvendo

**Comment by ruthtubau** [comment]
I'll merge when I have at least 1 👍, 🚢 , or :shipit:

**Comment by iservan** [comment]
👍  You can commit now 😛

**Comment by asidi** [comment]
👍

**Pull request merged**